packages/chafa.rb

```diff
@@ -0,0 +1,45 @@
+# Adapted from Arch Linux chafa PKGBUILD at:
+# https://github.com/archlinux/svntogit-community/raw/packages/chafa/trunk/PKGBUILD
+
+require 'package'
+
+class Chafa < Package
+  description 'Image-to-text converter supporting a wide range of symbols and palettes, transparency, animations, etc.'
+  homepage 'https://hpjansson.org/chafa/'
+  version '1.6.1'
+  license 'LGPL'
+  compatibility 'x86_64 aarch64 armv7l'
```

Comment:
Since this is a cli program, shouldn't this be 'all'?

Reply:
Yes, but it doesn't have binaries for i686. Feel free to add and modify...